tests/molecules-cells/pass_lee.js

```diff
@@ -0,0 +1,22 @@
+const complements = [
+  ['G', 'C'],
+  ['C', 'G'],
+  ['T', 'A'],
+  ['A', 'U'],
+]
+
+const RNA = (dna) =>
+  dna
+    .split('')
+    .map((ele) => Object.fromEntries(complements)[ele])
+    .join('')
+
+const DNA = (rna) =>
+  rna
+    .split('')
```

Comment:
@LEEDASILVA 
`split` convert a string into characters
`map` iterate over each of those

So why the for loop over a single character here ?